app/src/main/java/com/kickstarter/services/KSApolloClientV2.kt

```diff
@@ -236,18 +238,29 @@ class KSApolloClientV2(val service: ApolloClient) : ApolloClientTypeV2 {
         }
     }
 
-    override fun userPrivacy(): Observable<UserPrivacyQuery.Data> {
+    override fun userPrivacy(): Observable<UserPrivacy> {
         return Observable.defer {
-            val ps = PublishSubject.create<UserPrivacyQuery.Data>()
+            val ps = PublishSubject.create<UserPrivacy>()
             service.query(UserPrivacyQuery.builder().build())
                 .enqueue(object : ApolloCall.Callback<UserPrivacyQuery.Data>() {
                     override fun onFailure(exception: ApolloException) {
                         ps.onError(exception)
                     }
 
                     override fun onResponse(response: Response<UserPrivacyQuery.Data>) {
-                        response.data?.let {
-                            ps.onNext(it)
+                        response.data?.me()?.let {
+                            val defaultCurrency = "USD"
+                            val userPrivacy = UserPrivacy(
+                                name = it.name(),
+                                email = it.email() ?: "",
+                                hasPassword = it.hasPassword() ?: false,
+                                isCreator = it.isCreator ?: false,
+                                isDeliverable = it.isDeliverable ?: false,
+                                isEmailVerified = it.isEmailVerified ?: false,
+                                chosenCurrency = it.chosenCurrency() ?: defaultCurrency
```

Comment:
For non-mandatory fields, include a default value so that it is never null.

Reply:
Small suggestion, no need to change it if you don't see the need for it. but we could move this piece into `fun userPrivacyTransformer(UserPrivacyQuery.Data): UserPrivacy` within `GraphQLTransformers` and add some isolated unit tests on `GraphQLTransformersTest`

Reply:
for default currency here, same as below comment, let's use CurrencyCode.USD.rawValue()

Reply:
Yup! I had made same comment in your doc about moving logic to GraphQLTransformers. Will do.